code/modules/clothing/gloves/color.dm

```diff
@@ -7,6 +7,10 @@
 	permeability_coefficient = 0.05
 	item_color="yellow"
 
+/obj/item/clothing/gloves/color/yellow/fake
+	desc = "These don't feel like rubber..."

```

Comment:
You weren't gonna make it ```"These gloves will protect the wearer from electric shock. These don't feel like rubber..."```? Seem better as it also explain what they *should* do.